### PR TITLE
Language conversion

### PR DESCRIPTION
**Pull request by tschmidtb51**
- resolves https://github.com/oasis-tcs/csaf/issues/399
- add conversion rule for language
- adopt committee note

### CODE REVIEW

File: csaf_2.0/prose/csaf-v2-editor-draft.md [hunk 1]
```diff
@@ -5679,6 +5679,7 @@ Secondly, the program fulfills the following for all items of:
...
 * `/document/acknowledgments[]/organization` and `/vulnerabilities[]/acknowledgments[]/organization`: If more than one `cvrf:Organization` instance is given, the CVRF CSAF converter converts the first one into the `organization`. In addition, the converter outputs a warning that information might be lost during conversion of document or vulnerability acknowledgment.
+* `/document/lang`: If one or more CVRF element containing an `xml:lang` attribute exist and contain the exact same value, the CVRF CSAF converter converts this value into `lang`. If the values of `xml:lang` attributes are not equal, the CVRF CSAF converter outputs a warning that the language could not be determined and possibly a document with multiple languages was produced. In addition, it SHOULD also present all values of `xml:lang` attributes as a set.
```
Comment (sthagen) [inline csaf_2.0/prose/csaf-v2-editor-draft.md:5682]: @tschmidtb51 Could you kindly help me understand the additional requirement? Maybe it is obvious, but where should the converter present all values of the found language attributes as a set? Thanks a lot in advance.

**Review by sthagen — APPROVED** [review]

File: csaf_2.0/prose/csaf-v2-editor-draft.md (hunk 1 above)
Comment (tschmidtb51) [reply csaf_2.0/prose/csaf-v2-editor-draft.md:5682]: In the warning.

Comment (tschmidtb51) [reply csaf_2.0/prose/csaf-v2-editor-draft.md]: Hopefully clarified in [6b1d1f5](https://github.com/oasis-tcs/csaf/pull/428/commits/6b1d1f53b9f93bfebd179edfa5f363954bc7a5e2).

File: csaf_2.0/prose/csaf-v2-editor-draft.md [hunk 2]
```diff
@@ -5679,7 +5679,7 @@ Secondly, the program fulfills the following for all items of:
...
-* `/document/lang`: If one or more CVRF element containing an `xml:lang` attribute exist and contain the exact same value, the CVRF CSAF converter converts this value into `lang`. If the values of `xml:lang` attributes are not equal, the CVRF CSAF converter outputs a warning that the language could not be determined and possibly a document with multiple languages was produced. In addition, it SHOULD also present all values of `xml:lang` attributes as a set.
+* `/document/lang`: If one or more CVRF element containing an `xml:lang` attribute exist and contain the exact same value, the CVRF CSAF converter converts this value into `lang`. If the values of `xml:lang` attributes are not equal, the CVRF CSAF converter outputs a warning that the language could not be determined and possibly a document with multiple languages was produced. In addition, it SHOULD also present all values of `xml:lang` attributes as a set in the warning.
```
Comment (sthagen) [inline csaf_2.0/prose/csaf-v2-editor-draft.md:5682]: 👍🏽

**Pull request merged**